experimental/batchmap/tmap.go

```diff
@@ -16,9 +16,6 @@
 // verifiable data structures.
 package batchmap
 
-//go:generate go install github.com/apache/beam/sdks/go/cmd/starcgen
```

Comment:
Isn't this for the new stuff, is it still needed?

Reply:
Well spotted. This was supposed to be a temporary removal when I got into a bind with generated files. Somehow it jumped the sandbox and got into this PR. Reverted.